lib/msal-common/src/cache/entities/AccessTokenEntity.ts

```diff
@@ -76,7 +78,9 @@ export class AccessTokenEntity extends CredentialEntity {
         userAssertionHash?: string,
         keyId?: string,
         requestedClaims?: string,
-        requestedClaimsHash?: string
+        requestedClaimsHash?: string,
+        managedIdentityClientId?: string,
+        managedIdentityResource?: string
```

Comment:
I think you can avoid adding these 2 parameters and just pass managedIdentityClientId as clientId and managedIdentityResource as scopes. Should work fine to generate cache entity.